experimental/streaming/exporter/exporter.go

```diff
@@ -55,7 +55,7 @@ type Observer interface {
 const (
 	INVALID EventType = iota
 	START_SPAN
-	FINISH_SPAN
+	END_SPAN
```

Comment:
You changed the name of the EventType constant. We generate a file with strings based on those constant, so with this change, the file needs to be regerenerated. You can do it by running `make` - it will call the `go generate` and other stuff. With that done and commited, the CI should be fine.